diff --git a/node_modules/tinymce/tinymce.js b/node_modules/tinymce/tinymce.js
index 9687839..18f66fb 100644
--- a/node_modules/tinymce/tinymce.js
+++ b/node_modules/tinymce/tinymce.js
@@ -24590,6 +24590,7 @@
       scrollRangeIntoView(editor, rng);
     };
     var insertBrAtCaret = function (editor, evt) {
+console.log("@@@@@@ insertBrAtCaret");
       var selection = editor.selection;
       var dom = editor.dom;
       var rng = selection.getRng();
@@ -24632,11 +24633,13 @@
       editor.undoManager.add();
     };
     var insertBrBefore = function (editor, inline) {
+console.log("@@@@@ insertBrBefore");
       var br = SugarElement.fromTag('br');
       before(SugarElement.fromDom(inline), br);
       editor.undoManager.add();
     };
     var insertBrAfter = function (editor, inline) {
+console.log("@@@@@ insertBrAfter");
       if (!hasBrAfter(editor.getBody(), inline)) {
         after(SugarElement.fromDom(inline), SugarElement.fromTag('br'));
       }

